test/recipes/90-test_external.t

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env perl
+# Copyright 2015-2016 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+
+use OpenSSL::Test;
+use OpenSSL::Test::Utils;
+use OpenSSL::Test qw/:DEFAULT srctop_file/;
+
+setup("test_external");
+
+if (!$ENV{BORING_RUNNER_DIR}) {
```

Comment:
Nit pick: since you seem to want a fairly general test possibility, you might as well make the environment variable general as well...  like `EXTERNAL_RUNNER_DIR`